src/common/metrics/case_density.tsx

```diff
@@ -94,36 +95,42 @@ function renderStatus(projections: Projections): React.ReactElement {
     newCasesPerDay >= 0.1 && newCasesPerDay < 1
       ? formatDecimal(newCasesPerDay, 1)
       : formatInteger(newCasesPerDay);
-  const newCasesPerYear = 365 * newCasesPerDay;
-  const estimatedNewInfectionsPerYear =
-    ESTIMATED_INFECTIONS_FACTOR * newCasesPerYear;
+  const threeMonthsFromNow = moment().add(3, 'months');
+  const daysTilThreeMonthsFromNow = threeMonthsFromNow.diff(moment(), 'days');
+  const newCasesPerThreeMonths = daysTilThreeMonthsFromNow * newCasesPerDay;
+  const estimatedNewInfectionsPerThreeMonths =
+    ESTIMATED_INFECTIONS_FACTOR * newCasesPerThreeMonths;
 
   // Makes sure estimated # of cases and infections over the next year don't go above total population:
   function maxAtTotalPopulation(num: number) {
     return Math.min(num, totalPopulation);
   }
-  const estimatedNewInfectionsPerYearWithMax = maxAtTotalPopulation(
-    estimatedNewInfectionsPerYear,
+  const estimatedNewInfectionsPerThreeMonthsWithMax = maxAtTotalPopulation(
+    estimatedNewInfectionsPerThreeMonths,
+  );
+  const newCasesPerThreeMonthsWithMax = maxAtTotalPopulation(
+    newCasesPerThreeMonths,
   );
-  const newCasesPerYearWithMax = maxAtTotalPopulation(newCasesPerYear);
 
-  const estimatedPercentageNewInfectedPerYear = Math.min(
+  const estimatedPercentageNewInfectedPerThreeMonths = Math.min(
     1,
-    estimatedNewInfectionsPerYear / totalPopulation,
+    estimatedNewInfectionsPerThreeMonths / totalPopulation,
   );
 
   return (
     <Fragment>
       Over the last week, {locationName} has averaged {newCasesPerDayText} new
       confirmed cases per day (<b>{formatDecimal(currentCaseDensity, 1)}</b> for
-      every 100,000 residents). If this trend continued for the next year, this
-      would translate to approximately {formatEstimate(newCasesPerYearWithMax)}{' '}
+      every 100,000 residents). If this trend continued for the next three
+      months, by {threeMonthsFromNow.format('MMMM Do YYYY')} this would
```

Comment:
i'm not sure if we need to state the date. maybe just "if this trend continued for the next three months, this would translate to..."